.github/workflows/production-new-version.yml

```diff
@@ -0,0 +1,109 @@
+name: Production Release With Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (optional)'
+        required: false
+        default: ''
+      confirm:
+        type: boolean
+        required: true
+        default: 'false'
+        description: Yes, I want to release.
+
+jobs:
+  publish-production:
+    name: Publish to PRODUCTION 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Confirm
+        if: ${{ github.event.inputs.confirm != 'true' }}
+        run: |
+          echo "Please confirm that you want to release"
+          exit 1
+      - name: ⇣ Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_BUILDER_TOKEN }}
+          fetch-depth: 0
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - uses: actions/cache@v2
+        name: Cache Gradle packages
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Set version
+        run: |
+          set -e
+          OLD_VERSION="$(./gradlew -q currentVersion)"
+          NEW_VERSION="${{ github.event.inputs.version }}"
+          if [ -z "$NEW_VERSION" ]; then
+            NEW_VERSION=$(echo $OLD_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g' )
+          fi
+          sed -i "s/project\.externalVersion : '.*\..*\..*'/project\.externalVersion : '$NEW_VERSION'/g" build.gradle
+
+      - name: Get Version
+        id: get_build_version
+        run: |
+          VERSION="$(./gradlew -q currentVersion)"
+          echo "new version is $VERSION"
+          echo "::set-output name=build_version::$VERSION"
+
+      - name: Setup git
+        run: |
+          set -ex
+          git config --global user.email "builder+skip-ci@codota.com"
+          git config --global user.name "codota-builder"
+
+      - name: Commit changes
+        run: |
+          NEW_VERSION="${{ steps.get_build_version.outputs.build_version }}"
+          git pull origin master --no-edit
+          git commit -am "Set version to $NEW_VERSION"
+          git push
+          git tag v$NEW_VERSION
+          git push origin v$NEW_VERSION
+
+      - name: Create Tag
```

Comment:
why is it dupliacate?

Reply:
because i forgot to remove it of course 😬